src/js/subtraction/components/__tests__/__snapshots__/Subtraction.test.js.snap

```diff
@@ -50,5 +50,8 @@ exports[`<Subtraction /> should render 1`] = `
 exports[`<SubtractionFileManager /> should render 1`] = `
 <Connect(FileManager)
   fileType="subtraction"
+  message="Drag FASTA file(s) here to upload."
+  tip="Accepts .fa and .fasta files with or without gzip compression."
```

Comment:
1. "Drag FASTA files here to upload"
2. "Accepts files ending in fa, fasta, fa.gz, or fasta.gz." (easier to understand for lab tech)